libs/designsystem/accordion/src/accordion-item.component.spec.ts

```diff
@@ -72,4 +80,12 @@ describe('AccordionItemComponent', () => {
       });
     });
   });
+  it('should emit the toggle event when collapsed', () => {
+    spyOn(spectator.component.toggle, 'emit');
+    spectator.component.isDisabled = false;
+    spectator.component.isExpanded = true;
+    spectator.component._onToggleExpanded(new KeyboardEvent('keydown'));
+    expect(spectator.component.toggle.emit).toHaveBeenCalledTimes(1);
+    expect(spectator.component.toggle.emit).toHaveBeenCalledWith(false);
```

Comment:
Consider readability - separate code blocks in [arrange-act-assert](https://medium.com/@pjbgf/title-testing-code-ocd-and-the-aaa-pattern-df453975ab80):
```suggestion
    spyOn(spectator.component.toggle, 'emit');
    spectator.component.isDisabled = false;
    spectator.component.isExpanded = true;
    
    spectator.component._onToggleExpanded(new KeyboardEvent('keydown'));
    
    expect(spectator.component.toggle.emit).toHaveBeenCalledTimes(1);
    expect(spectator.component.toggle.emit).toHaveBeenCalledWith(false);
```